pages/api/v1/activity-feed/explore-posts.ts

```
@@ -0,0 +1,373 @@
+				tally,
+				timeline: proposalTimeline,
+				title: subsquareTitle || 'Untitled',
+				topic: topicFromType,
+				totalVotes: totalVotes || 0,
+				track_no: !isNaN(trackNumber) ? trackNumber : null,
+				type: type
+			};
+
+			await getContentSummary(post, network, true);
+
+			delete post?.content;
```

Biome

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)